Documentation/building/unix-instructions.md

```diff
@@ -21,50 +95,33 @@ cd corefx
 ```bash
 managed            # optional argument to build the managed code
 native             # optional argument to build the native code
+platform           # OS to compile for (FreeBSD, Linux, NetBSD, OSX, Windows)
+skiptests          # build, but do not run, the managed unit tests
+BuildType          # build configuration type (Debug, Release)
 
 # The following arguments affect native builds only:
 
 BuildArch          # build architecture (x64, x86, arm, arm64)
-BuildType          # build configuration type (Debug, Release)
 clean              # optional argument to force a clean build
 verbose            # optional argument to enable verbose build output
 clangx.y           # optional argument to build using clang version x.y
-platform           # OS to compile for (FreeBSD, Linux, NetBSD, OSX, Windows)
 cross              # optional argument to signify cross compilation, uses ROOTFS_DIR environment variable if set
-skiptests          # skip the tests in the './bin/*/*Tests/' subdirectory
 cmakeargs          # user-settable additional arguments passed to CMake
-```
-
-### Prerequisites
-
-* git
-* curl-dev (libcurl4-openssl-dev)
-* icu (libicu52)
-* cmake
-* clang (clang-3.5)
-* libunwind (libunwind8)
-
-> Note: These instructions have been validated on:
-* Ubuntu 15.04, 14.04, and 12.04
-* Fedora 22
-* OS X 10.10 (Yosemite)
-* FreeBSD 10.2
-* NetBSD 7.0
-* Alpine Linux 3.3
 
+```
 ### Known Issues
 If you see errors along the lines of `SendFailure (Error writing headers)` you may need to import trusted root certificates:
 
 ```sh
 mozroots --import --sync
 ```
 
-## Steps to Build on Ubuntu
+## Steps to build on Ubuntu 14.04 LTS
```

Comment:
My only hesitation here is that we're being too explicit with the Ubuntu version numbers.  I think it's ok to note what it has been tested on, but have we tried these builds steps on Ubuntu 16.04?  I think it would be beneficial to the public if we could make a statement like `Ubuntu 14.04+` and get blanket coverage on the newer builds of Ubuntu.